docs/GAME_FLOW.md

```diff
@@ -141,76 +142,59 @@ various pieces of global behaviour.
       <code>enable_tr2_item_drops</code>
     </td>
     <td>Boolean</td>
-    <td>No</td>
     <td>
       Forces enemies who are placed in the same position as pickup items to
       carry those items and drop them when killed, similar to TR2+. See
       <a href="#item-drops">Item drops</a> for full details.
     </td>
   </tr>
   <tr valign="top">
-    <td>
-      <a name="enforced-config"></a>
-      <code>enforced_config</code>
-    </td>
+    <td><a name="enforced-config"></a>
+    <code>enforced_config</code></td>
     <td>String-to-object map</td>
-    <td>No</td>
     <td>
       This allows <em>any</em> regular game config setting to be overriden. See
       <a href="#user-configuration">User configuration</a> for full details.
     </td>
   </tr>
   <tr valign="top">
-    <td>
-      <code>injections</code>
-    </td>
+    <td><code>injections</code></td>
     <td>String array</td>
-    <td>No</td>
     <td>
       Global data injection file paths. Individual levels will inherit these
       unless <code>inherit_injections</code> is set to <code>false</code> on
       those levels. See <a href="#injections">Injections</a> for full details.
     </td>
   </tr>
   <tr valign="top">
-    <td>
-      <code>levels</code>
-    </td>
-    <td>Object array</td>
-    <td>Yes</td>
+    <td><code>levels</code></td>
+    <td>Object array<strong>*</strong></td>
```

Comment:
Are some of these `Type`s supposed to have the `*` at the end?

Reply:
Yes as they're required as explained in disclaimer below.